docs/configuration.rst

```diff
@@ -68,8 +68,25 @@ below:
    * - ``DD_TRACE_DEBUG``
      - Boolean
      - False
-     - Enables debug logging in the tracer. Setting this flag will cause the library to create a root logging handler if
-       one does not already exist. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_DEBUG``).
+     - Enables debug logging in the tracer. Setting this flag will cause the library to create a root logging handler if one does not already exist. Added in ``v0.41.0`` (formerly named ``DATADOG_TRACE_DEBUG``). Can be used with `DD_TRACE_LOG_FILE` to route logs to a file.
+
+       .. _dd-trace-log-file-level:
+   * - ``DD_TRACE_LOG_FILE_LEVEL``
+     - String
+     - DEBUG
+     - Configures the ``RotatingFileHandler`` used by the `ddtrace` logger to write logs to a file based on the level specified. Defaults to `DEBUG`, but will accept the values found in the standard **logging** library, such as WARNING, ERROR, and INFO, if further customization is needed. Files are not written to unless ``DD_TRACE_LOG_FILE`` has been defined.
+
+       .. _dd-trace-log-file:
+   * - ``DD_TRACE_LOG_FILE``
+     - String
+     - None
+     - Directs `ddtrace` logs to a specific file. Note: The default backup count is 1. For larger logs, use with `DD_TRACE_LOG_FILE_SIZE_BYTES`. To fine tune the logging level, use with ``DD_TRACE_LOG_FILE_LEVEL``. 
+
+       .. _dd-trace-log-file-size-bytes:
+   * - ``DD_TRACE_LOG_FILE_SIZE_BYTES``
+     - Int
+     - 15728640
+     - Max size for a file when used with `DD_TRACE_LOG_FILE`.
```

Comment:
Maybe a note here about the backup count is valuable, so you know the max is actually 2 * `DD_TRACE_LOG_FILE_SIZE_BYTES`, what do you think?

Reply:
Good point! I just adjusted the configuration text to 

>Max size for a file when used with `DD_TRACE_LOG_FILE`. When a log has exceeded this size, there will be one backup log file created. In total, the files will store ``2 * DD_TRACE_LOG_FILE_SIZE_BYTES`` worth of logs.

Let me know if this is what you were thinking.

Reply:
Just added to the latest commit - let me know if the new changes work!

```
    file_log_level_value = logging.DEBUG
    try:
        file_log_level_value = getattr(logging, log_file_level.upper())
    except AttributeError:
        raise ValueError(
            "Unknown log level. Logging level must be CRITICAL/ERROR/WARNING/INFO/DEBUG. Detected logging level was ",
            str(log_file_level),
        )
```